generators/express/templates/base/.gitignore

```
@@ -1,3 +1,5 @@
 /coverage/
 /dist/
 /node_modules/
+yarn-error.log
+yarn-debug.log
```

I personally prefer to only ignore files in the current folder by prefixing the ignore path with `/`:

```suggestion
/yarn-error.log
/yarn-debug.log
```